scripts/run_e2e_tests.py

```diff
@@ -265,22 +311,61 @@ def run_tests(args):
                 'PORTSERVER_ADDRESS': common.PORTSERVER_SOCKET_FILEPATH,
             }))
 
-        stack.enter_context(servers.managed_webdriver_server(
-            chrome_version=args.chrome_driver_version))
+        if args.suite == 'full':
+            stack.enter_context(servers.managed_webdriver_server(
+                chrome_version=args.chrome_driver_version))
+
+            proc = stack.enter_context(servers.managed_protractor_server(
+                suite_name=args.suite,
```

Comment:
Is the above functioning(not sure though) or you need to implement something like below? Ditto for webdriverio?
```suggestion
                suite_name=args.SUITES_STILL_IN_PROTRACTOR,
```

Reply:
Yes the function works, while running the full suite there will be 4 instance of e2e running : 3 for protractor tests and 1 for webdriverio. Also we cannot pass the list of suite to webdriverio.

Reply:
I am not able to understand the context of "there will be 4 instance of e2e running : 3 for protractor tests and 1 for webdriverio.". Can you please ellaborate?

Reply:
I meant [this](https://github.com/oppia/oppia/wiki/End-to-End-tests#configure-test-sharding). If we are currently running all the suites at once, we will be running 3 instance of protractor tests and 1 instance of webdriverio.

Reply:
I understood for 3 protractor tests but I think there should be three for webdriveio too, right?

Reply:
But that will result in total 6 instances which is not recommended. Once protractor will be removed I will add 3 instance for webdriverio.

Reply:
Can you please specify the code that enables only 1 instance for webdrierio?

Reply:
Default value is 1 if we have provided only one browser name as capabilities in webdriverio.